core/graph/src/main/java/uk/gov/gchq/gaffer/operation/export/graph/handler/ExportToOtherGraphHandler.java

```diff
@@ -0,0 +1,179 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.operation.export.graph.handler;
+
+import uk.gov.gchq.gaffer.graph.Graph;
+import uk.gov.gchq.gaffer.operation.export.graph.ExportToOtherGraph;
+import uk.gov.gchq.gaffer.operation.export.graph.OtherGraphExporter;
+import uk.gov.gchq.gaffer.store.Context;
+import uk.gov.gchq.gaffer.store.Store;
+import uk.gov.gchq.gaffer.store.StoreProperties;
+import uk.gov.gchq.gaffer.store.library.GraphLibrary;
+import uk.gov.gchq.gaffer.store.operation.handler.export.ExportToHandler;
+import uk.gov.gchq.gaffer.store.schema.Schema;
+import uk.gov.gchq.koryphe.ValidationResult;
+
+public class ExportToOtherGraphHandler extends ExportToHandler<ExportToOtherGraph, OtherGraphExporter> {
+    private static final String ID = "gaffer.store.id";
+
+    @Override
+    protected Class<OtherGraphExporter> getExporterClass() {
+        return OtherGraphExporter.class;
+    }
+
+    @Override
+    protected OtherGraphExporter createExporter(final ExportToOtherGraph export, final Context context, final Store store) {
+
+        return new OtherGraphExporter(
+                context.getUser(),
+                context.getJobId(),
+                createGraph(export, store));
+    }
+
+    protected Graph createGraph(final ExportToOtherGraph export, final Store store) {
+
+        final String exportGraphId = export.getGraphId();
+        final Schema exportSchema = export.getSchema();
+        final StoreProperties exportStoreProperties = export.getStoreProperties();
+        final String exportParentSchemaId = export.getParentSchemaId();
+        final String exportParentStorePropertiesId = export.getParentStorePropertiesId();
+
+        final String storeGraphId = store.getGraphId();
+        final GraphLibrary storeLibrary = store.getGraphLibrary();
+        final StoreProperties storeStoreProperties = store.getProperties();
+        final Schema storeSchema = store.getSchema();
+
+        ValidationResult validationResult = validate(exportGraphId, exportParentSchemaId, exportParentStorePropertiesId,
+                storeLibrary, exportSchema, exportStoreProperties, store);
+        if (!validationResult.isValid()) {
+            throw new IllegalArgumentException(validationResult.getErrorString());
+        }
+
+        StoreProperties storeProperties;
+        Schema schema;
+
+        // No store graph library so we create a new Graph
+        // Im pretty sure this isnt possible, storeLibrary will never be null as it is set in the graph.Builder?
+        if (null == storeLibrary) {
```

Comment:
It is unlikely to be null, but if a Store implementation overrides it, it could be. Perhaps just leave this in just in case. Please can you update your comment?